index.html

```diff
@@ -4,43 +4,193 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio Main Page</title>
+    <title>Grid and Flexbox</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container">
-        <header class="box header">
-            <nav>
-                <span><a href="index.html">My Logo</a></span>
-                <span class ="hamburger"><img src="img/Enabled.png"></span>
-            </nav>
-        </header>
-        <section class="box content">
-            <h1 class="title">
-            I’m Mariane 
-            Glad to see 
-            you!
-            </h1>
-            <p class="paragraph">
-                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
-                   If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            
-            <div class="connect">
+    <div class="grid-container">
+        <div class="grid-item flex-container">
+            <div class ="flex-item flex-item-1"><img class="deneme" src="../img/Project-1.png"></div>
+            <div class ="flex-item flex-item-2"><h1 class="project-content-2">Tonic</h1></div>
+            <div class ="flex-item flex-item-3">
```

Comment:
### See below image for a guide on how file path works in HTML.
![image](https://user-images.githubusercontent.com/84156714/192067991-78f4fb55-c494-4860-87d2-de27f2b425f3.png)

- [ ] [REQUIRED] The images all over your project appear broken. I see that the image path you used seems to be the issue. Use `With "img/image_name.png"` instead of `With "../img/image_name.png"`. Take note of the double ..

With "../img/image_name.png" | With "img/image_name.png"
---------- | ------
![image](https://user-images.githubusercontent.com/84156714/192067478-6b08ed94-769f-471f-aafd-bc60d01209fb.png) | ![image](https://user-images.githubusercontent.com/84156714/192067565-2f82db4d-9225-4998-b996-0e86dfd994bb.png)